cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramNotificationSubscriberService.java

```diff
@@ -385,7 +385,7 @@ private void handleProgramEvent(ProgramRunId programRunId, ProgramRunStatus prog
         writeToHeartBeatTable(recordedRunRecord, logicalStartTimeSecs, programHeartbeatTable);
         runRecordMonitorService.removeRequest(programRunId, true);
         long startDelayTime = logicalStartTimeSecs - RunIds.getTime(programRunId.getRun(), TimeUnit.SECONDS);
-        emitStartingTimeMetric(programRunId, startDelayTime, recordedRunRecord);
+        emitStartingTimeMetric(programRunId, startDelayTime, programRunStatus, recordedRunRecord);
```

Comment:
This metric does not need to emitted when the pipeline transitions to `RUNNING` state now otherwise there will be 2 metrics emitted for same pipeline run - 
1. When the pipeline was in `RUNNING` state.
2. When the pipeline gets completed i.e reaches `END_STATE`.